devicon.json

```diff
@@ -142,6 +157,9 @@
     },
     {
         "name": "androidstudio",
+        "altnames": [
+            "arm64"
```

Comment:
why is androidstudio also named "arm64"? It's also a duplicate of "aarch64". This brings me to the point of: Should we allow duplicates for altnames? I would say "no", otherwise they would be used as tags, not alternative names. Or? So we would need a check for duplicate altnames.

Reply:
ohh, yes. Must have been a merge issue since I accidentally based the branch on master at first, then rebased on develop.

I will fix this soon :)

Reply:
fixed: 3c5bf839850823623296b0f0a222ba65aff6386c